webapp/app/translations/de/LC_MESSAGES/messages.po

```diff
@@ -5128,6 +5128,34 @@ msgstr ""
 "zuständigen Finanzamt beantragt. Die neue Steuernummer wird Ihnen dann "
 "mit dem Steuerbescheid mitgeteilt."
 
+#: app/templates/unlock_code/activation_failure.html:15
+msgid "form.unlock-code-activation.failure.reasons-title"
+msgstr "Mögliche Ursachen:"
+
+#: app/templates/unlock_code/activation_failure.html:17
+msgid "form.unlock-code-activation.failure.reasons-1"
+msgstr ""
+"Ihr Freischaltcode ist nicht korrekt. Eine Ursache kann die Verwechslung "
+"von Ziffern oder Buchstaben sein, die sich ähneln. So kann die Ziffer 0 "
+"schnell mit dem Buchstaben O oder der Buchstabe B schnell mit der Ziffer "
+"8 verwechselt werden. Prüfen Sie Ihre Angabe."
+
+#: app/templates/unlock_code/activation_failure.html:18
+msgid "form.unlock-code-activation.failure.reasons-2"
+msgstr ""
+"Haben Sie sich vor dem 03.07.2021 registriert? In diesem Fall können Sie "
+"sich mit diesem Freischaltcode nicht mehr beim Steuerlotsen anmelden. "
+"Registrieren Sie sich bitte erneut. Sie erhalten dann einen Brief mit "
+"einem neuen Freischaltcode."
```

Comment:
Can we add a reminder in the Backlog for the 1st of October to change this to "vor 90 Tageb" or to "today - 90 days" in the call of this text?

Reply:
Good idea